### ISSUE TITLE
ECR path is incorrect @ main.yaml

### ISSUE THREAD

**Issue by aravindh-c** [yadav-avnish/aps-fault-detection#3]
aps-fault-detection/.github/workflows/main.yaml at 

1)Line no.73

( @ ` docker pull ${{secrets.AWS_ECR_LOGIN_URI}}/${{ secrets.ECR_REPOSITORY_NAME }}:latest) `

Invalid path : ` ${{secrets.AWS_ECR_LOGIN_URI}}/${{ secrets.ECR_REPOSITORY_NAME }}:latest `
Valid path : ` ${{secrets.AWS_ECR_LOGIN_URI}}:latest`

2) @ Line No.76

invalid path : ` export IMAGE_NAME="${{secrets.AWS_ECR_LOGIN_URI}}"/"${{ secrets.ECR_REPOSITORY_NAME }}":latest AWS_ACCESS_KEY_ID="${{ secrets.AWS_ACCESS_KEY_ID }}" AWS_SECRET_ACCESS_KEY="${{ secrets.AWS_SECRET_ACCESS_KEY }}" AWS_DEFAULT_REGION=${{ secrets.AWS_REGION }} BUCKET_NAME=${{ secrets.BUCKET_NAME }} "MONGO_DB_URL=${{ secrets.MONGO_DB_URL }}" && docker compose up -d `

valid path : `export IMAGE_NAME="${{secrets.AWS_ECR_LOGIN_URI}}":latest AWS_ACCESS_KEY_ID="${{ secrets.AWS_ACCESS_KEY_ID }}" AWS_SECRET_ACCESS_KEY="${{ secrets.AWS_SECRET_ACCESS_KEY }}" AWS_DEFAULT_REGION=${{ secrets.AWS_REGION }} BUCKET_NAME=${{ secrets.BUCKET_NAME }} "MONGO_DB_URL=${{ secrets.MONGO_DB_URL }}" && docker compose up -d`

**Comment by digvijaytaunk** [comment]
@aravindh-c, The fully qualified path to pull the docker image should be `docker pull 93XXXXXXX582.dkr.ecr.ap-south-1.amazonaws.com/aps-image-name:latest`. In my opinion, it totally depends upon the value you set in the variable `secrets.AWS_ECR_LOGIN_URI`. I believe the URI and image name are set in 2 different variables by Avnish. And hence this path.

**Comment by aravindh-c** [comment]
I agree with you. That's why I corrected the path based on the current
implementation.
